nuget/readme.md

```diff
@@ -59,6 +59,19 @@ C#/WinRT behavior can be customized with these project properties:
 * -output $(CsWinRTGeneratedFilesDir)
 * $(CsWinRTFilters)
 
+## Advanced size saving options
+
+C#/WinRT supports several feature switches to opt-out of some features available by default, like projection of built-in .NET types. These features are often required by applications, but might be unnecessary in scenarios such as minimal WinRT components authoring. In those cases, it's possible to use these properties to opt-out of any of these features, which can provide binary size savings and performance improvements.
+
+| Property | Value(s) | Description |
+|-|-|-|
+| CsWinRTEnableDynamicObjectsSupport | \*true \| false | Enables support for marshalling WinRT types to dynamic objects (ie. `ExpandoObject`). Setting this to **false** allows all of `System.Linq.Expressions` to be trimmed out. |
+| CsWinRTUseExceptionResourceKeys | true \| \*false | Uses minimal, non-localized exception messages to save binary size. Setting this to **true** allows stripping all of the localization .resw files that are bundled as embedded resources into C#/WinRT. |
+| CsWinRTEnableDefaultCustomTypeMappings | \*true \| false | Enables all available custom type mappings of built-in .NET types by default (eg. `INotifyPropertyChanged`). Setting this to **false** allows trimming all code requires to support those features. |
```

Comment:
Do we want to mention this: https://github.com/microsoft/CsWinRT/blob/staging/AOT/src/WinRT.Runtime/Projections.CustomTypeMappings.g.cs

Reply:
Done! 👍